bin/heudiconv

```diff
@@ -272,7 +273,15 @@ def process_dicoms(fl, dcmsession=None, basedir=None, dcmfilter=None):
             refphys = dcminfo.ReferringPhysicianName
         except AttributeError:
             refphys = '-'
-
+        if dcminfo.get([0x18,0x24], None):
+            # GE and Philips scanners
```

Comment:
Styling: Regular variables shouldn't be capitalized as classes. Make it eg `sequence_name `